lib/fastlane/plugin/mango/helper/mango_helper.rb

```diff
@@ -27,6 +28,7 @@ def initialize(params)
         @pull_latest_image = params[:pull_latest_image]
 
         @docker_commander = DockerCommander
+        @emulator_commander = EmulatorCommander
```

Comment:
Is there a reason to have instance variables of those modules? You can just use the module directly.